app/Http/Controllers/Api/LinkController.php

```diff
@@ -31,11 +31,6 @@ public function store(StoreLink $request)
         // Get the user ID so we can attach it to the link
         $user_id = ($request->user()->id) ? $request->user()->id : null;
```

Comment:
this line really doesn't do anything, as you are guaranteed to have this value ... i would just add `$request->user()->id` at the point where you call $user_id